corehq/apps/domain/management/commands/reset_user_passwords.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from bulk_update.helper import bulk_update as bulk_update_helper
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+
+from corehq.apps.users.models import CommCareUser
+from corehq.util.view_utils import absolute_reverse
+from dimagi.utils.chunked import chunked
+from six.moves import input
+
+
+class Command(BaseCommand):
+    help = "Reset all mobile user passwords to a given password"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'domain_name',
+        )
+        parser.add_argument(
+            'password',
+        )
+
+    def handle(self, domain_name, password, **options):
+        url = absolute_reverse('domain_homepage', args=[domain_name])
+        confirm = input(
+            """
+            ## Caution!! ##
+            ###############
+            You are resetting passwords of all users in {domain} at {url}
+            Are you sure want to reset, type your domain name to confirm?
+            """.format(domain=domain_name, url=url)
+        )
+        if not confirm == domain_name:
+            print("Aborted")
+            return
+
+        all_users = CommCareUser.by_domain(domain_name)
+        django_users = User.objects.filter(
+            username__endswith="@{domain}.commcarehq.org".format(domain=domain_name)
+        )
+        assert len(all_users) == django_users.count(), "Something is wrong, django/couch users don't match"
+
+        num_users = len(all_users)
+        for i, users in enumerate(chunked(all_users, 1000)):
+            for user in users:
+                user.set_password(password)
+            CommCareUser.bulk_save(users)
+            print("Updated {count}/{total}".format(count=(i+1)*1000, total=num_users))
+
+        for i, users in enumerate(chunked(django_users, 1000)):
+            for user in users:
+                user.set_password(password)
```

Comment:
it looks like set_password creates a new password hash. could this be different on the couch and django user and if so does that matter? i imagine there is some sort of random salt added

Reply:
It's the same, couch `set_password` actually pulls from Django

Reply:
right but that function generates a random salt each time, so if we use it once on the couch user and once on the django user the hashed strings will be different. unless im missing something?

Reply:
Ha, that's a good point! The `check_password` still works based on raw-password, I also tested this locally on a domain and trying to login to make sure everything works.

Alternatively, we could generate the hash once for all users and use the same for all users.

Reply:
if it still works and its only for testing purposes here that is good enough for me. just wanted to make sure we were testing since it seemed like that could have been a problem

Reply:
Cool, I am not favorite of this either, but for throwaway test data, it seems okay.